app/src/js/utils/table-config/requests.js

```diff
@@ -32,18 +35,13 @@ export const getPrivileges = () => {
       isStaff: privileges.find(o => o.match(/ADMIN/g))
         ? privileges.find(o => o.match(/ADMIN/g))
         : roles.find(o => o.short_name.match(/staff/g)),
-      canReview: privileges.find(o => o.match(/ADMIN/g))
```

Comment:
There is existing logic in `utils/privileges.js` to figure out what request privileges are available. In order to reduce duplication, I've called `requestPrivileges` here. That brings along the logic for determining if someone can REVIEW or REVIEW_MANAGER